TVShows/TVShows/Modules/ShowDetails/Cells/ShowDetailsEpisodeCell.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  ShowDetailsEpisodeCell.swift
+//  TVShows
+//
+//  Created by Ivan Milicevic on 22/10/2018.
+//  Copyright © 2018 Ivan Milicevic. All rights reserved.
+//
+
+import UIKit
+
+class ShowDetailsEpisodeCell: UITableViewCell {
+
+    @IBOutlet weak var episodeTitleLabel: UILabel!
+    @IBOutlet weak var seasonEpisodeLabel: UILabel!
```

Comment:
Same as with properties, you can (and should) make IBOutlets private and prefix them with underscore.

`@IBOutlet private weak var _episodeTitleLabel: UILabel!`